gcp/v2/asm/istio-operator.yaml

```diff
@@ -25,15 +25,14 @@ spec:
       istio-ingressgateway:
         type: NodePort
     global:
-      #meshID: "jlewi-dev_us-central1_kf-bp-0420-002" # {"type":"string","x-kustomize":{"setBy":"kpt","partialSetters":[{"name":"gcloud.core.project","value":"jlewi-dev"},{"name":"name","value":"kf-bp-0420-002"},{"name":"location","value":"us-central1"}]}}
-      meshID: "jlewi-dev_us-central1_kf-bp-0420-002" # 
+      meshID: "jlewi-dev_us-central1_kf-bp-0420-002" # {"type":"string","x-kustomize":{"setBy":"kpt","partialSetters":[{"name":"gcloud.core.project","value":"jlewi-dev"},{"name":"name","value":"kf-bp-0420-002"},{"name":"location","value":"us-central1"}]}}
       trustDomain: "jlewi-dev.svc.id.goog" # {"type":"string","x-kustomize":{"partialSetters":[{"name":"gcloud.core.project","value":"jlewi-dev"}]}}
       sds:
         token:
           aud: "jlewi-dev.svc.id.goog" # {"type":"string","x-kustomize":{"partialSetters":[{"name":"gcloud.core.project","value":"jlewi-dev"}]}}
       proxy:
         env:
-          GCP_METADATA: "jlewi-dev|147474701642|asm-cluster|us-central1" # {"type":"string","x-kustomize":{"partialSetters":[{"name":"gcloud.core.project","value":"jlewi-dev"},{"name":"gcloud.project.projectNumber","value":"147474701642"},{"name":"name","value":"asm-cluster"},{"name":"gcloud.compute.zone","value":"us-central1-c"}]}}
+          GCP_METADATA: "jlewi-dev|147474701642|asm-cluster|us-central1-c" # {"type":"string","x-kustomize":{"partialSetters":[{"name":"gcloud.core.project","value":"jlewi-dev"},{"name":"gcloud.project.projectNumber","value":"147474701642"},{"name":"name","value":"asm-cluster"},{"name":"gcloud.compute.zone","value":"us-central1-c"}]}}
```

Comment:
Note in comment, "value" is "us-central1-c". This fix makes sure kpt cfg set can update us-central1-c.

Reply:
Can you update https://github.com/kubeflow/manifests/blob/6ef44e866ec362c2a0d1d612d0dcc4551cb6a78a/tests/Makefile#L31 to reset these values?

Reply:
@jlewi Sure, but I'd want that to be a separate PR.
This one is just meant fo fix bugs.